js/apps/system/_admin/aardvark/APP/frontend/js/views/userView.js

```diff
@@ -104,12 +104,21 @@
         )
       );
 
-      buttons.push(
-        window.modalView.createNotificationButton(
-          'Change Password',
-          this.editUserPassword.bind(this)
-        )
-      );
+      if (this.username.substring(0, 6) === ':role:') {
+        buttons.push(
+          window.modalView.createDisabledButton(
+            'Change Password',
+            this.editUserPassword.bind(this)
```

Comment:
seems superfluous to pass "editUserPassword" here, as the button is disabled and the `createDisabledButton` function does not have two arguments in its signature.